tools/install/Makefile.skel

```diff
@@ -65,6 +65,7 @@ FORT_LIB     = ""
 endif
 endif
 
+## set compilation flags for each compiler
```

Comment:
Something in these changes causes the unit tests to fail on Travis, but they work locally. Any clue?

Reply:
L1307 at https://travis-ci.org/AtChem/AtChem2/jobs/561733054 indicates that it's not the Makefile, but rather than Fruit isn't being installed, because the download is failing. Looks like they've got an expired certificate. I would hope this will be fixed in the next few days :/

Reply:
Ok, so it is their fault 😀 

What do you think of these changes? Do they make sense? I also wanted to add a bit more comments to the `Makefile` for future reference but I am not sure about all the sections.

Reply:
I've made review - just a couple of things really, the rest looks good and useful.